foundations/foundation-test-scaffolding/src/main/java/io/vertx/core/impl/SyncContext.java

```diff
@@ -16,12 +16,6 @@
  */
 package io.vertx.core.impl;
 
-import java.io.Serial;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ConcurrentMap;
-import java.util.concurrent.Executor;
-import java.util.concurrent.Executors;
```

Comment:
why change this import order?

Reply:
okay，I have cancelled the optimization import